tests/src/Traits/GeneratedContentTestMockTrait.php

```diff
@@ -80,64 +78,6 @@ protected static function getProtectedValue(object $object, string $property) {
     return $property->getValue($class);
   }
 
-  /**
```

Comment:
@AlexSkrypnyk 
I have removed this prepare class mock because we are not using and 
https://github.com/sebastianbergmann/phpunit/issues/5241